src/components/AddItem/AddItem.jsx

```diff
@@ -1,3 +1,168 @@
+import styles from "./AddItem.module.scss";
+import { useState, useRef } from "react";
+
 export default function AddItem() {
-  return <div>addItem</div>;
+  const [imagePreview, setimagePreview] = useState(null);
+  const [formData, setFormData] = useState({
+    image: "",
+    name: "",
+    about: "",
+    price: "",
+    tag: [],
+  });
+  const inputRef = useRef();
+  const active = Object.values(formData).every((value) =>
+    Array.isArray(value) ? value.length > 0 : value
+  );
+
+  function handleChange(e) {
+    const { name, value } = e.target;
+    if (name === "image") {
+      const newImage = e.target.files[0];
+      setimagePreview(URL.createObjectURL(newImage));
+      setFormData((prev) => ({ ...prev, image: newImage }));
+    } else {
+      setFormData((prev) => ({ ...prev, [name]: value }));
+    }
+  }
+  function handleTag(e) {
+    if (e.key === "Enter") {
+      e.preventDefault(); //폼 내부에서 엔터 -> 자동 제출되는거 막음
+      setTimeout(() => {
+        const newValue = inputRef.current.value.trim();
+        if (newValue) {
+          setFormData((prev) => ({
+            ...prev,
+            tag: [...prev.tag, newValue],
+          }));
+        }
+        inputRef.current.value = "";
+      }, 0);
+    }
+  }
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(formData); // 서버 전송 등 처리
+  };
+
+  console.log(formData);
+  function handleRemoveTag(tagToRemove) {
+    setFormData((prev) => ({
+      ...prev,
+      tag: prev.tag.filter((tag) => tag !== tagToRemove),
+    }));
+  }
+
+  return (
+    <>
+      <form className={styles.form} onSubmit={handleSubmit}>
+        <header className={styles["form__header"]}>
+          <h1 className={styles.h1}>상품 등록하기</h1>
+          <button
+            className={`${styles["submit--active"]} ${
+              active ? "" : styles["submit--disabled"]
+            }`}
+            type="submit"
+            disabled={!active}
+          >
+            등록
+          </button>
+        </header>
+        <div className={styles["form__body"]}>
+          <div className={styles["input__image"]}>
+            <label
+              htmlFor="input-image"
+              className={`${styles["input__label"]}  ${styles["image--input"]}`}
+            >
+              상품 이미지
+            </label>
+            <input
+              id="input-image"
+              type="file"
+              accept="image/png, image/gif, image/jpeg"
+              name="image"
+              onChange={handleChange}
+              placeholder="이미지 등록"
+              style={{ display: "none" }}
+            />
+            {formData.image && (
+              <div className={styles["image--preview"]}>
+                <img src={imagePreview} alt="미리보기" />
+                <button
+                  className={`${styles.xButton} ${styles["delete-image"]}`}
```

Comment:
❗️ 수정요청
button type 속성의 기본값은 submit입니다~ type을 button으로 명시해주세요!

```suggestion
                <button
                  type="button"
                  className={`${styles.xButton} ${styles["delete-image"]}`}
```